src/plugins/npm/npmPlugin.js

```diff
@@ -20,12 +21,7 @@ module.exports = async function npmPlugin(msg) {
 
   msg.handling();
 
-  if (!/^[a-zA-Z0-9_.@/~-]{3,}$/.test(name)) {
-    msg.respondWithMention(`that doesn't look like a valid package name`);
-    return;
-  }
-
-  if (name.startsWith('~')) {
+  if (name.startsWith('*')) {
```

Comment:
I’m a bit confused here actually is ~ special in the current bot? Like `!npm ~f` searches instead of looks up?

in that case maybe a question mark would be better?

Reply:
currently it is yes, but I noticed `~` is allowed as a first char, so I changed it to `*` which is not. It's used to perform a search instead of a direct name lookup

Reply:
ok, question mark is indeed better